config_generator/src/main.rs

```diff
@@ -0,0 +1,191 @@
+// Copyright 2021 Google LLC
```

Comment:
Should this be under `cloudflare_worker/config_generator` since it is Cloudflare-specific?

Reply:
There is a possibility that in the future the Fastly config is also generated by this code.